task_1.py

```diff
@@ -1,2 +1,13 @@
-def cross(n):
-    pass
+#CROSS
```

Comment:
Hej, pisanie komentarzy w kodzie to jest trochę przynanie się do porażki (chociaż czasem ciężko jest to zrobić inaczej) - kod powinień sam tłumaczyć swoje działanie, dodawanie w komentarzu nazwy funkcji obok faktycznej nazwy funkcji to raczej niedobry nawyk.